conf/uppmax.config

```diff
@@ -9,25 +9,48 @@ singularity {
   enabled = true
 }
 
+def hostname = "sinfo --local -N -h | head -1 | cut -f1 -d' ' ".execute().text.trim()
+
 process {
+  // closure to create a suitable clusterOptions
+  def clusterOptionsCreator = { m ->
```

Comment:
I'd be inclined to keep this out of the `process` scope, alongside `hostname` instead

Reply:
Thanks, fixed.